<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sketchfab Uploader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Model name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leModelName"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteDescription"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lTags">
        <property name="text">
         <string>Tags:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leTags"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbPrivate">
        <property name="text">
         <string>Private</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lPassword">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePassword">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="bUploadToSketchfab">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Upload to Sketchfab</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Model URL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lModelURL">
        <property name="text">
         <string/>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSketchfab_settings"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_Sketchfab_Uploader">
   <property name="text">
    <string>About Sketchfab Uploader</string>
   </property>
  </action>
  <action name="actionSketchfab_settings">
   <property name="text">
    <string>Sketchfab settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
